static/src/javascripts/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import { isInVariant } from 'common/modules/experiments/utils';
+import { abTestClashData as acquisitionsAbTestClashData } from 'common/modules/experiments/acquisition-test-selector';
+
+const emailTests: Object[] = [];
+const contributionsTests: Object[] = acquisitionsAbTestClashData;
```

Comment:
This type should really be `ABTest[]`, so that `emailTests` and `potentiallyClashingTests` could also be `ABTest`. Since `acquisitionsAbTestClashData` has type `AcquisitionsABTest` which is a subtype of `ABTest` I thought I might be able to do this - but normal arrays [are invariant in Flow](https://stackoverflow.com/questions/45483050/flowtype-extend-array-type-parameter) because they are mutable.

**But...!** Flow does actually have a [`$ReadOnlyArray`](https://github.com/facebook/flow/blob/18a237e2aa3712ad3566c2f9fc1a530dea2666b9/lib/core.js#L199-L239) type which is covariant. I'm going to convert `ABTest[]`, `AcquisitionsABTest[]` etc to use this type but since this will inevitably involve some extra refactoring I'll leave it for a separate PR